label_studio/server.py

```diff
@@ -306,6 +306,15 @@ def main():
         calculate_stats_all_orgs(input_args.from_scratch, redis=True)
         return
 
+    if input_args.command == 'export':
+        from tasks.functions import export_project
+        filename = export_project(
+            input_args.project_id, input_args.export_format, input_args.path
+        )
+
+        print(f'Project exported successfully: {filename}')
```

Comment:
![image](https://user-images.githubusercontent.com/501780/185272870-fa823b3f-e805-4942-a067-7b7f60b08027.png)

Reply:
looks not friendly..